4-nodarbiba/uzdevumi/MadaraBalana/objects/js/app.js

```diff
@@ -1,3 +1,48 @@
+let madaraBalana = {
+  firstName: "Madara",
+  lastName: "Balana",
+  email: "madara.balanas@gmail.com",
+
+  telNr: 26164785,
+
+  workExperience: [
+    "Saeimas Administrācijas Saeimas sēžu nodrošinājuma nodaļa",
+    "SIA Palink",
+    "Valsts sociālās apdrošināšanas aģentūra",
+    "Studijas :)",
+  ],
+
+  languages: {
+    lv: {
+      name: "Latviešu valoda",
+      level: "Native",
+    },
+    eng: {
+      name: "Angļu valoda",
+      level: "Proficient",
+    },
+    rus: {
+      name: "Krievu valoda",
+      level: "Intermediate",
+    },
+    de: {
+      name: "Vācu valoda",
+      level: "Intermediate",
+    },
+  },
+};
+
+console.log(
+  "Personas vārds, uzvārds: ",
+  madaraBalana.firstName + " " + madaraBalana.lastName
```

Comment:
Šis, protams, strādā un ir ok.
Bet labi būtu izmantot "string literals" sintaksi:

`'${madaraBalans.firstName} ${madaraBalans.lastName}'`